### PR TITLE
[RPS-90] Add "Resources Links" to publication

### PR DESCRIPTION
**Pull request by wojtek-oledzki**
... and small CSS/templates update.

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/configuration/translations/cms.yaml [hunk 1]
```diff
@@ -9,3 +9,4 @@ definitions:
         without a value. Please remove the placeholder or select a value.'
       publicationsystem-blank-related-link: There is a related link field with no
         URL provided. Please remove it or enter a URL.
+      valid-url: Valid link must be of format http://... or https://...
```
Comment (ivorc) [inline repository-data/application/src/main/resources/hcm-config/configuration/translations/cms.yaml]: Suggest rewording to "Link URL must start with http:// or https://"

File: repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftl [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
-This data set is part of <a class="label label--parent-document"
-    href="<@hst.link hippobean=parentPublication.selfLinkBean/>"
-    title="${parentPublication.title}"
->${parentPublication.title}</a>
-
-<div class="content-section">
+<div class="content-section content-section--highlight">
     <h2 class="doc-title" data-uipath="ps.dataset.title">${dataset.title}</h2>
 
+    This data set is part of <a class="label label--parent-document"
```
Comment (ivorc) [inline repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftl]: "data set" should be one word, I think: "dataset"

File: repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl [hunk 3]
```diff
@@ -34,8 +34,22 @@
...
+    <div class="content-section content-section--highlight">
+
+        <div data-uipath="ps.publication.information-types">
+            <#list publication.informationType as type><#if type?index != 0>, </#if>${type?html}</#list>
+        </div>
+
         <h2 class="doc-title" data-uipath="ps.publication.title">${publication.title?html}</h2>
+
+        <#if parentSeries??>
+        This is part of
```
Comment (ivorc) [inline repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl]: Should this be "This publication is part of"?

Comment (wojtek-oledzki) [reply repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/publication.ftl]: no, agreed on plain "This is part of" with Martin, Peter, ...

Going forward, this will be a resource in CMS (`hst.setBundle`)
